scripts/build-storybook.js

```diff
@@ -0,0 +1,14 @@
+const dotenv = require("dotenv");
+const fs = require("fs");
+const { execSync } = require("child_process");
+
+const envConfig = dotenv.parse(fs.readFileSync(".env"));
```

Comment:
**suggestion**: I don't think you need `fs` here; you can just do `require('dotenv').config()` to load everything in from the `.env` file ([docs](https://www.npmjs.com/package/dotenv#%EF%B8%8F-usage))

Reply:
We don't want to load everything from the env file though